cli/cli_handler.cpp

```diff
@@ -83,7 +83,7 @@ void handleIssueESDT(cxxopts::ParseResult const &result)
             esdtProperties
     )->buildSigned(keyFile->getSeed());
 
-    auto const txHash = proxy.send(tx).hash;
+    auto const txHash = proxy.send(tx);
```

Comment:
For future - there's also an endpoint that allows one to send multiple transactions: `/transaction/send-multiple` (erdpy uses it, erdjs does not - yet).

Reply:
Good point, could add a task for this, thanks! :+1: